### ISSUE TITLE
[Feature Request]: Warn user about missing feature

### ISSUE THREAD

**Issue by przepompownia** [puremourning/vimspector#399]
I found (by inspecting the code of both sides) that vscode-php-debug does not support variable completion yet. The documentation of this adapter does not mention explicitly about it. On the other side Vimspector silently returns an empty result in such case.

Please warn user about missing feature (echomsg with echohl for example). 

https://github.com/puremourning/vimspector/blob/4e04a862cb37105acebac8b6ac5b275dc7865815/python3/vimspector/debug_session.py#L631-L632

**Comment by przepompownia** [comment]
What do you think about creating a separate command to display summary of supported and unsupported features for given debug adapter?

**Comment by przepompownia** [comment]
I didn't notified by GitHub about #406 before it was merged. 

Thanks. Info about `supportsCompletionsRequest` still remains missing.
```
--------------------------------------------------------------------------------
Server Capabilities: 
--------------------------------------------------------------------------------
{
  "supportsConfigurationDoneRequest": true,
  "supportsEvaluateForHovers": false,
  "supportsConditionalBreakpoints": true,
  "supportsFunctionBreakpoints": true,
  "exceptionBreakpointFilters": [
    {
      "filter": "Notice",
      "label": "Notices"
    },
    {
      "filter": "Warning",
      "label": "Warnings"
    },
    {
      "filter": "Error",
      "label": "Errors"
    },
    {
      "filter": "Exception",
      "label": "Exceptions"
    },
    {
      "filter": "*",
      "label": "Everything",
      "default": true
    }
  ],
  "supportTerminateDebuggee": true
}
--------------------------------------------------------------------------------
```

**Comment by puremourning** [comment]
Missing means ‘false’

**Comment by przepompownia** [comment]
There is no `"supportsCompletionsRequest": false`. I guess this capability isn't reported by PHP XDebug adapter explicitly.

**Comment by puremourning** [comment]
Exactly. If the entry is not there, it means the adapter doesn't support it.

**Comment by przepompownia** [comment]
Less advanced users (and me in case of possibly other capabilities) can be still confused as to why, for example, completion does not work (without browsing the code of Vimspector and the adapter). 

It would be good to list all the capabilities supported by Vimspector.

**Comment by puremourning** [comment]
it's a bunch of work that is quite low value imo, but I've added it to the backlog anyway
